spec/SideBar.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow } from 'enzyme';
+
+import SideBar from '../client/src/SideBar.jsx';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('About Component', () => {
+  test('should render correctly', () => {
+    expect(shallow(<SideBar />)).toMatchSnapshot();
+  });
+});
+
+describe("rendered `About`", () => {
+  it("received two props", () => {
```

Comment:
test description doesn't match the expect (which is looking for 4 props)

Reply:
updated test